### ISSUE TITLE
Blank screen when clicking "ADVANCED SETTINGS" tab on "datetime" field in collection type

### ISSUE THREAD

**Issue by aldomeza-dev** [strapi/strapi#18573]
### Required System information

- Environment: development
- OS: win32-x64
- Node.js version: v18.15.0 win32 x64 workspaces/false
- NPM version: 9.6.7
- Strapi version: 4.14.5 Community
- Database: MySQL 8.0 (in Docker, Docker version 20.10.22, build 3a2c30b)
- Operating system: Windows 11
- Is your project Javascript or Typescript: Javascript 

### Describe the bug

When adding a new **datetime** field to a collection type and clicking the **ADVANCED SETTINGS** tab, the screen goes blank and the console shows this:
![image](https://github.com/strapi/strapi/assets/87845126/01ba8606-616d-48fb-9891-b477d7353e35)

### Steps to reproduce the behavior

1. Create a new strapi project with MySQL 8.0
2. Create a new collection type
3. Add a **datetime** field to said collection type.
4. While creating (or after it's created) click the ADVANCED SETTINGS tab.
5. See error

### Expected behavior

To be able to see and interact with the ADVANCED SETTINGS tab of the datetime field I'm trying to configure. 

### Screenshots

![image](https://github.com/strapi/strapi/assets/87845126/e46474e0-fe36-4997-9b5e-9416df7ea38e)
![image](https://github.com/strapi/strapi/assets/87845126/8ccccb5f-97e7-4c23-852b-5b3e10723750)

### Code snippets

Docker compose code for the sabe DB as me:
```

version: "3"

services:
  factusync-db-container:
    image: mysql:8.0
    command: --default-authentication-plugin=mysql_native_password
    environment:
      MYSQL_ROOT_PASSWORD: GD%p&#3Cv78mY!%p
      MYSQL_DATABASE: factusync-db
      MYSQL_USER: factusync-db-admin
      MYSQL_PASSWORD: GD%p&3Cv78mY!%p
    ports:
      - "3306:3306"
    volumes:
      - mysql-data:/var/lib/mysql

volumes:
  mysql-data:
```

### Additional context

I've seen two issues that have been fixed and added to stable versions #18123 &  #18353.

**Comment by faceless49** [comment]
same error

**Comment by joshuaellis** [comment]
Have you tried upgrading to 4.15.0?

**Comment by Boegie19** [comment]
closing this issue since he is using 4.14.5 and this issue is fixed in 4.15.0
